sklearn/tests/test_dummy.py

```diff
@@ -101,6 +100,23 @@ def test_most_frequent_and_prior_strategy():
                                       clf.class_prior_.reshape((1, -1)) > 0.5)
 
 
+def test_most_frequent_and_prior_strategy_with_pandas_dataframe():
+    X = [[0], [0], [0], [0]]  # ignored
```

Comment:
I am not sure what you mean by this comment (maybe that X is irrelevant for this issue). In any case I think just remove the comment.